### ISSUE TITLE
Speech Paper Voting Round 2

### ISSUE THREAD

**Issue by EverlynAsiko** [masakhane-io/masakhane-reading-group#12]
In this issue you can either:

- Add papers that you think are interesting to read and discuss (please stick to the format).
- **Vote**: should be done using 👍 on comments

Example: [Voting Paper #1](https://github.com/masakhane-io/masakhane-reading-group/issues/1)

I have added some papers collected on the [Papers to read sheet](https://docs.google.com/spreadsheets/d/1yYsIlBstwy3WQ9yX2uXQnJMCsm-0P46JJLPfKRNzE8E/edit?usp=sharing)

**Comment by EverlynAsiko** [comment]
# Deep Speech: Scaling up end-to-end speech recognition

[Link to Paper](https://arxiv.org/pdf/1412.5567.pdf)

_Abstract_

We present a state-of-the-art speech recognition system developed using end-to-end deep learning. Our architecture is significantly simpler than traditional speech systems, which rely on laboriously engineered processing pipelines; these traditional systems also tend to perform poorly when used in noisy environments. In contrast, our system does not need hand-designed components to model background noise, reverberation, or speaker variation, but instead directly learns a
function that is robust to such effects. We do not need a phoneme dictionary, nor even the concept of a “phoneme.” Key to our approach is a well-optimized RNN training system that uses multiple GPUs, as well as a set of novel data synthesis techniques that allow us to efficiently obtain a large amount of varied data for training. Our system, called Deep Speech, outperforms previously published results on the widely studied Switchboard Hub5’00, achieving 16.0% error on the full test set. Deep Speech also handles challenging noisy environments better than widely used, state-of-the-art commercial speech systems.

**Comment by EverlynAsiko** [comment]
# Listen, Attend and Spell

[Link to paper](https://arxiv.org/abs/1508.01211)

_Abstract_

We present Listen, Attend and Spell (LAS), a neural network that learns to transcribe speech utterances to characters. Unlike traditional DNN-HMM models, this model learns all the components of a speech recognizer jointly. Our system has two components: a listener and a speller. The listener is a pyramidal recurrent network encoder that accepts filter bank spectra as inputs. The speller is an attention-based recurrent network decoder that emits characters as outputs. The network produces character sequences without making any independence assumptions between the characters. This is the key improvement of LAS over previous end-to-end CTC models. On a subset of the Google voice search task, LAS achieves a word error rate (WER) of 14.1% without a dictionary or a language model, and 10.3% with language model rescoring over the top 32 beams. By comparison, the state-of-the-art CLDNN-HMM model achieves a WER of 8.0%.

**Comment by EverlynAsiko** [comment]
# Automatic speech recognition: a survey

[Link to paper](https://www.researchgate.net/publication/345710977_Automatic_speech_recognition_a_survey/link/5fbc7321299bf104cf6e85c0/download)

_Abstract_

Recently great strides have been made in the field of automatic speech recognition (ASR) by using various deep learning techniques. In this study, we present a thorough comparison between cutting-edged techniques currently being used in this area, with a special focus on the various deep learning methods. This study explores different feature extraction methods, state-of-the-art classification models, and vis-a-vis their impact on an ASR. As deep learning techniques are very data-dependent different speech datasets that are available online are also discussed in detail. In the end, the various online toolkits, resources, and language models that can be helpful in the formulation of an ASR are also proffered. In this study, we captured every aspect that can impact the performance of an ASR. Hence, we speculate that this work is a good starting point for academics interested in ASR research.

**Comment by JRMeyer** [comment]
# Sequence Transduction with Recurrent Neural Networks

> The transducer described in this paper extends CTC by defining a distribution over output sequences of all lengths, and by jointly modelling both input-output and output-output dependencies.

[link to paper](https://arxiv.org/pdf/1211.3711.pdf)

*Abstract*

Many machine learning tasks can be expressed as the transformation — or _transduction_ — of input sequences into output sequences: speech recognition, machine translation, protein secondary structure prediction and text-to-speech to name but a few. One of the key challenges in sequence transduction is learning to represent both the input and output sequences in a way that is invariant to sequential distortions such as shrinking, stretching and translating. Recurrent neural networks (RNNs) are a powerful sequence learning architecture that has proven capable of learning such representations. However RNNs traditionally require a pre-defined alignment between the input and output sequences to perform transduction. This is a severe limitation since finding the alignment is the most difficult aspect of many sequence transduction problems. Indeed, even determining the length of the output sequence is often challenging. This paper introduces an end-to-end, probabilistic sequence transduction system, based entirely on RNNs, that is in principle able to transform any input sequence into any finite, discrete output sequence. Experimental results for phoneme recognition are provided on the TIMIT speech corpus.
